lean/src/valid/algebra_amgm_faxinrrp2msqrt2geq2mxm1div2x.lean

```diff
@@ -35,17 +35,17 @@ begin
   },
   rw [h₂, ←nnreal.coe_le_coe, nnreal.coe_of_real, nnreal.coe_add, nnreal.coe_mul, nnreal.coe_mul, nnreal.coe_of_real, nnreal.coe_of_real] at h₀,
   
-  have h₃ : 2 * ((1 / 2) ^ ((1:ℝ) / 2)) ≤ 2 * (↑((1:nnreal) / 2) * x + ↑((1:nnreal) / 2) * (1 / (2 * x))), {
+  have h₃ : 2 * ((1 / 2)^((1:ℝ) / 2)) ≤ 2 * (↑((1:nnreal) / 2) * x + ↑((1:nnreal) / 2) * (1 / (2 * x))), {
     refine (mul_le_mul_left _).mpr _,
     linarith,
     exact h₀,
   },
 
-  have h₄ : 2 * ((1 / 2) ^ ((1:ℝ) / 2)) = real.sqrt 2, {
+  have h₄ : 2 * ((1 / 2)^((1:ℝ) / 2)) = real.sqrt 2, {
     rw [eq_comm, real.sqrt_eq_iff_mul_self_eq],
-    calc (2:ℝ) * (1 / (2:ℝ)) ^ (1 / (2:ℝ)) * ((2:ℝ) * (1 / (2:ℝ)) ^ (1 / (2:ℝ))) = (2:ℝ) * (2:ℝ) * ( (1 / (2:ℝ)) ^ (1 / (2:ℝ)) *  (1 / (2:ℝ)) ^ (1 / (2:ℝ))) : by {ring,}
-      ... = (2:ℝ) * (2:ℝ) * (1 / (2:ℝ)) ^ ((1 / (2:ℝ)) + (1 / (2:ℝ))) : by {rw real.rpow_add, linarith,}
-      ... = (2:ℝ) * (2:ℝ) * (1 / (2:ℝ)) ^ (1:ℝ) : by {congr', apply add_halves,}
+    calc (2:ℝ) * (1 / (2:ℝ))^(1 / (2:ℝ)) * ((2:ℝ) * (1 / (2:ℝ))^(1 / (2:ℝ))) = (2:ℝ) * (2:ℝ) * ( (1 / (2:ℝ))^(1 / (2:ℝ)) *  (1 / (2:ℝ))^(1 / (2:ℝ))) : by {ring,}
```

Comment:
might have messe with alignment of calc here?

Reply:
I guess no, I was more worried about the alignment of the calc lines ?

Reply:
yeah, at least lean doesn't complain about it.